INTLY-4388: Rewrite to remove ambiguous language (#126)
integr8ly/tutorial-web-app-walkthroughs: walkthroughs/1B-protecting-applications-using-rh-sso/walkthrough.adoc

.. Paste the URL of the *Order Entry System UI* from the *Integrating message-oriented middleware with a RESTful API using AMQ Online* Solution Pattern in the *Root URL* field. This should look similar to `https://order-entry-ui-{user-username}-<NAMESPACE>.{openshift-app-host}`
.. Click *Save*.
. The *Settings* screen for the `order-entry-system` client should be displayed.
. Verify that the *Access Type* field is set to `public`. This means the client is a frontend application that needs to log in using a web browser.

[type=verification]
Select *Clients* in the side menu. 
Is the `order-entry-system` client listed and is the *Enabled* field set to `True`?

[type=verificationFail]

> Verify that you followed each step in the procedure above. In the *Settings* tab on the *Client* page, the *Enabled* toggle should be set to *ON*. If you are still having issues, contact your administrator.

Reviewer: ```suggestion
Verify that you followed each step in the procedure above. In the *Settings* tab on the *Client* page, verify that the *Enabled* toggle is set to *ON*. If you are still having issues, contact your administrator.
```
